pyproject.toml

```diff
@@ -100,8 +100,63 @@ count = true
 arg-type-hints-in-docstring = false
 
 [tool.pyright]
-# pyright 1.1.340 and newer hangs up when analyzing this file.
 exclude = [
-    "./src/orquestra/sdk/_base/_traversal.py"
+    "src/orquestra/sdk/_base/_driver/_client.py",
```

Comment:
ooft. Excluding this many files may cause us problems if we never go back and fix the issues.

Did you enumerate the issues somewhere (like Slack or Confluence)?

Reply:
Listed: https://zapatacomputing.atlassian.net/wiki/spaces/ORQSRUN/pages/842105015/Pyright+Type+Errors

Reply:
At the moment we're ignoring pyright errors for all files. You can treat my PR as _enabling_ type checks for all the remaining files not listed in the excludelist.

Reply:
IMHO I think this is amazing strategy to move forward. We are enabling check for "what works for now". And later during either by ticket-by-ticket or during innovation time or whatever we can start removing exceptions one by one basically.